### ISSUE TITLE
Maximum execution time of 0 seconds exceeded in ... emailcloak.php on line 489

### ISSUE THREAD

**Issue by h-k-h** [joomla/joomla-cms#15818]
I had this problem accessing apparently any page in my website: After significant time, may be 30" a blank page showed. After enabling maximum error reporting the following message was displayed instead of a blank page:
Fatal error: Maximum execution time of 0 seconds exceeded in /mnt/web123/b0/44/53538444/htdocs/joomla_01/plugins/content/emailcloak/emailcloak.php on line 489 
- however the administration worked fine!

The problem could be reproduced with MS-Internet-Explorer (Windows 10) and Firefox (windows 7). 
Being under pressure to restore access to the website I decided to install Email Protector v4.1.1 
Fortunately the problem was resolved for me.

### Steps to reproduce the issue
unknown. 

### Expected result
pages according to defined content

### Actual result
After significant time, may be 30" a blank page showed.
After enabling maximum error reporting the following message was displayed instead of a blank page:
Fatal error: Maximum execution time of 0 seconds exceeded in /mnt/web123/b0/44/53538444/htdocs/joomla_01/plugins/content/emailcloak/emailcloak.php on line 489 

### System information (as much as possible)

Systeminformationen
Einstellung 	Wert
 
PHP erstellt für 	SunOS localhost 5.10 Generic_142901-13 i86pc
Datenbankversion 	5.6.36-log
Datenbankzeichensatz 	latin1_german1_ci
Database Connection Collation 	utf8mb4_general_ci
PHP-Version 	5.6.30
Webserver 	Apache/2.2.31 (Unix)
PHP-Interface für den Webserver 	cgi-fcgi
Joomla!-Version 	Joomla! 3.6.4 Stable [ Noether ] 21-October-2016 16:33 GMT
Joomla!-Plattform-Version 	Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
Browsererkennung 	Mozilla/5.0 (Windows NT 6.1; WOW64; rv:53.0) Gecko/20100101 Firefox/53.0 

### Additional comments
In error_log_20170430-20170504.txt I found lots of messages like the following (after April 28):
04.05.2017 18:07:54 grossauheimer-geschichtsverein.de [client 2a01:238:e100::] PHP Fatal error:  Maximum execution time of 0 seconds exceeded in /mnt/web123/b0/44/53538444/htdocs/joomla_01/plugins/content/emailcloak/emailcloak.php on line 489
Access log revealed a break down of the website.

**Comment by brianteeman** [comment]
does the email address include any special characters such as umlauts?

**Comment by h-k-h** [comment]
Apparently all the pages of the web site were affected, I tested only a few. I am pretty sure that the content articles have no email addresses at all. But I have no idea if something surrounding my content to make up the http page has something of that kind in it or at least a string that might be confused with an email address. All the email addresses I am aware of are in the contacts but I did not even test with the contacts page. I would not know any address containing strange characters such as umlaut, the problem is that there might be such a thing despite I am not aware of... Have you a means to verify e.g. our welcome page for such suspected strings or suggest a procedure for me to do so?
